libs/base/ui/action/color-mode-selector/src/color-mode-selector.component.ts

```diff
@@ -77,7 +77,7 @@ export class ColorModeSelectorComponent extends LitElement {
           aria-label="${i18n('site.change-color-mode')}"
           @click=${this.triggerEvent}
         >
-          <oryx-icon type="${iconType}"></oryx-icon>
+          <oryx-icon type=${iconType}></oryx-icon>
```

Comment:
```suggestion
          <oryx-icon .type=${iconType}></oryx-icon>
```

(we do not use the dot syntax in stories, as it won't show the attribute in the documentation tab)